taskt/Core/Automation/Commands/Error Handling Commands/GetExceptionMessageCommand.cs

```diff
@@ -1,57 +1,60 @@
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Output String Variable")]
+        [InputSpecification("Choose a variable to take the output error string.")]
+        [SampleUsage("vSomeVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required " +
+                 "to pre-define your variables, however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
 
         public GetExceptionMessageCommand()
         {
-            this.CommandName = "GetExceptionMessageCommand";
-            this.SelectionName = "Get Exception Message";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "GetExceptionMessageCommand";
+            SelectionName = "Get Exception Message";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
+
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
             var error = engine.ErrorsOccured.OrderByDescending(x => x.LineNumber).FirstOrDefault();
-            var errorMessage = engine.FileName + " Line " + error.LineNumber.ToString() + "Error: " + error.ErrorMessage;
-            errorMessage.StoreInUserVariable(sender, v_userVariableName);
-
+            var errorMessage = engine.FileName + " Line " + error.LineNumber + "Error: " + error.ErrorMessage;
+            errorMessage.StoreInUserVariable(sender, v_OutputUserVariableName);
         }
+
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
-            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_userVariableName", this));
-            var VariableNameControl = CommandControls.CreateStandardComboboxFor("v_userVariableName", this).AddVariableNames(editor);
-            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_userVariableName", this, new Control[] { VariableNameControl }, editor));
-            RenderedControls.Add(VariableNameControl);
+            RenderedControls.AddRange(
+                CommandControls.CreateDefaultOutputGroupFor("v_OutputUserVariableName", this, editor)
+                );
 
             return RenderedControls;
         }
+
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [Store in: '" + v_userVariableName + "']";
+            return base.GetDisplayValue() + $" [Store String In: {v_OutputUserVariableName}]";
```

Comment:
"in" Should be lowercase without the colon. I changed it in the template a few days ago. Cameron was right. Also this could be more descriptive. I didn't mean for TYPE to be so literal. Maybe "Store Exception Message in..."

Reply:
Done